api.yaml

```diff
@@ -14,284 +14,80 @@ externalDocs:
-      tags:
-        - lifecycle-events
-      operationId: 'org.zalando.stups.yourturn.api/read-lifecycle-events-by-application'
-      parameters:
-        - $ref: '#/parameters/ApplicationID'
-        - name: start_date
-          in: query
-          description: date specifying to only return events that occured past (inclusive) that date.
-          required: false
-          type: string
-          format: date-time
-          allowEmptyValue: false
-        - name: end_date
-          in: query
-          description: date specifying to only return events that occured before (inclusive) that date.
-          required: false
-          type: string
-          format: date-time
-          allowEmptyValue: false
-        - name: eventnames
-          in: query
-          description: coma separated list of event names so that only events excatly matching one of theese names is returned
-          required: false
-          type: string
-          format: string
-          allowEmptyValue: false
-        - name: sort
-          in: query
-          description: specifying the sorting on event names or change date when an event occured
-          required: false
-          type: string
-          format: string
-          allowEmptyValue: false
-      responses:
-        200:
-          description: Returns an object with application and version ID and a an array of requested data
-          schema:
-            type: object
-            properties:
-              application_id:
-                type: string
-                description: Unique identifier of the application
-                example: kio
-              version_id:
-                type: string
-                description: Unique identifier of the version
-                example: somestring
-              data:
-                type: array
-                items:
-                   type: object
-                   properties:
-                     change_date:
-                       type: string
-                       format: date-time
-                       description: Point in time when the instance count of the version changed.
-                       example: '2015-04-25T16:25:00.000Z'
-                     event_name:
-                       type: string
-                       description: name of the occured event
-                       example: 'someEventThatHappened'
 
-        404:
-          description: Not found e.g. no valid application ID or version ID or both
-        default:
-          $ref: '#/responses/Error'
-
-  '/lifecycle/events/apps/{application_id}/versions/{version_id}':
+  '/servercount/{application_id}':
```

Comment:
I would call it "instance-count", because we have the notion "instance" here at Zalando since beginning of the time (deployctl) and it also matches the AWS wording.